Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +179,18 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        items['Penalty'] = self.PENALTIES[self.penalty_type]
+        items['Learning rate'] = self.LEARNING_RATES[self.learning_rate]
+        items['Constants'] = "Alpha={}, Eta0={}, Epsilon={},\
+                              L1 ratio={}, Power_t={}".format(self.alpha,
+                              self.eta0, self.epsilon, self.l1_ratio,
+                              self.power_t)
```

Comment:
@janezd would probably have this formatted differently. Not all options are applicable in all cases, i.e. L1 ratio only makes sense with elasticnet penalty.